mindsdb/integrations/libs/ml_exec_base.py

```diff
@@ -147,6 +148,100 @@ def learn_process(class_path, context_dump, integration_id,
     db.session.commit()
 
 
+@mark_process(name='adjust')
+def adjust_process(class_path, context_dump, integration_id,
```

Comment:
Maybe worth refactoring the common bits between this process and learn into helper(s) function(s)?